storage/leveldb/leveldbSerial.go

```diff
@@ -28,15 +28,16 @@ type SerialDB struct {
 
 // NewSerialDB is a constructor for the leveldb persister
 // It creates the files in the location given as parameter
-func NewSerialDB(path string, batchDelaySeconds int, maxBatchSize int) (s *SerialDB, err error) {
+func NewSerialDB(path string, batchDelaySeconds int, maxBatchSize int, maxOpenFiles int) (s *SerialDB, err error) {
 	err = os.MkdirAll(path, rwxOwner)
 	if err != nil {
 		return nil, err
 	}
```

Comment:
Should there be a verification against a minimum possible number for `maxOpenFiles`? Would `leveldb` work well when `OpenFilesCacheCapacity = 0`?

Reply:
Will test to see the appropriate boundaries.

Reply:
Done.